src/docs/product/issues/issue-details/performance-issues/n-one-api-calls.mdx

```diff
@@ -15,8 +15,9 @@ Sentry detects N+1 API Calls in frontend and mobile applications. The detector i
 The N+1 API Call detector looks for groups of simultaneous network calls to the same URL. It uses the following detection criteria:
 
 - Applies to `GET` requests only
-- Minimum of 10 network calls must be happening simultaneously
-- Network calls must be a minimum of 50ms in duration and within 5ms of each other
+- Minimum of `10` network calls must be happening simultaneously
+- Total duration of involved spans must exceed `300ms`.
```

Comment:
Comparing this doc to the other two, it looks like this one might be missing the note that you can configure this threshold on the Project Settings > Performance page?

Reply:
I think you can configure N+1 DB queries but not N+1 API calls, but would be good to confirm.